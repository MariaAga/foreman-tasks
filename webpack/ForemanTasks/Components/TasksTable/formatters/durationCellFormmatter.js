import React from 'react';

export const durationCellFormmatter = value => (
  <span className="param-value" title={value.tooltip}>
    {value.text}
  </span>
);
