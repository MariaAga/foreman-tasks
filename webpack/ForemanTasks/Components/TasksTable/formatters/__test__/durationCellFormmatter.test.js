import { durationCellFormmatter } from '../durationCellFormmatter';

describe('durationCellFormmatter', () => {
  it('render with tooltip', () => {
    expect(
      durationCellFormmatter({ text: 'some-value', tooltip: 'some-tooltip' })
    ).toMatchSnapshot();
  });
  it('render without tooltip', () => {
    expect(durationCellFormmatter({ text: 'some-value' })).toMatchSnapshot();
  });
});
