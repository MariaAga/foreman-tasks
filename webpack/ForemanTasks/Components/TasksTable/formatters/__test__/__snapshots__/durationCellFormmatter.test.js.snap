// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`durationCellFormmatter render with tooltip 1`] = `
<span
  className="param-value"
  title="some-tooltip"
>
  some-value
</span>
`;

exports[`durationCellFormmatter render without tooltip 1`] = `
<span
  className="param-value"
>
  some-value
</span>
`;
